src/transport/msg_types/pause.py

```diff
@@ -38,6 +39,7 @@
 class PauseMessage(TypedDict):
     type: str
     blockNumber: int
+    blockHash: Hash32
```

Comment:
We don't need it to setup protocol on pause. Remove it

Reply:
Removed